src/scenes/AcceleratorRouter/Modules/EventsCellRenderer.tsx

```diff
@@ -23,6 +23,22 @@ export default function EventsCellRenderer(props: RendererProps<TableRowType>):
     return <CellRenderer index={index} column={column} value={valueToRender} row={row} />;
   }
 
+  if ((column.key === 'meetingUrl' || column.key === 'recordingUrl' || column.key === 'slidesUrl') && !onRowClick) {
+    return (
+      <CellRenderer
+        index={index}
+        column={column}
+        value={
+          <Link fontSize='16px' target='_blank' to={value as string}>
+            {' '}
+            {value as string}{' '}
+          </Link>
```

Comment:
I assume the rendered spaces are intentional to achieve some desired spacing?

Reply:
Oh, no, thanks for noticing this.

Reply:
🙌